db/migrations/versions/018_fix_pfc_table.py

```diff
@@ -0,0 +1,40 @@
+"""empty message
+
+Revision ID: 018_fix_pfc_table
+Revises: 017_add_pfc_table
+Create Date: 2024-03-07 16:30:09.404394
+
+"""
+
+import sqlalchemy as sa
+from alembic import op
+
+# revision identifiers, used by Alembic.
+revision = "018_fix_pfc_table"
+down_revision = "017_add_pfc_table"
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("project_finance_change", schema=None) as batch_op:
+        batch_op.alter_column("programme_junction_id", existing_type=sa.UUID(), nullable=False)
+        batch_op.drop_index("ix_project_finance_change_join_programme_junction")
+        batch_op.create_index(
+            "ix_project_finance_change_join_programme_junction", ["programme_junction_id"], unique=False
+        )
+
+    # ### end Alembic commands ###
+
+
+def downgrade():
+    # ### commands auto generated by Alembic - please adjust! ###
+    with op.batch_alter_table("project_finance_change", schema=None) as batch_op:
+        batch_op.drop_index("ix_project_finance_change_join_programme_junction")
+        batch_op.create_index(
+            "ix_project_finance_change_join_programme_junction", ["programme_junction_id"], unique=False
```

Comment:
I presume this is auto-generated? Any idea why it recreates what looks like exactly the same index?

Reply:
I'm also confused as to why it isn't re-created after the column is changed - this is why I presumed it was recreated

Reply:
ah, so this downgrade should actually set unique = True? And should be placed after the column is edited?